src/sass/layout/_main.scss

```diff
@@ -0,0 +1,520 @@
+@use '../abstracts';
+@use './functions' as *;
+
+.icons-section {
+
+    $ideal-icons-section-height: 600;
+    $iconssection-ideal-sidepadding: 340;
+    $ideal-icons-section-paddingbottom: 100;
+    $container-margin-bottom: 6.5rem;
+    $sticker-width: 15.125rem;
+    $sticker-height: 14.375rem;
+    $icon-width: 6.875rem;
+    $icon-height: 6.25rem;
+    $title-padding-bottom: 1.5rem;
+
+    height: #{scaleHeight($ideal-icons-section-height)};
+    width: inherit;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding-left: #{scaleWidth($iconssection-ideal-sidepadding)};
+    padding-right: #{scaleWidth($iconssection-ideal-sidepadding)};
+    background-color: var(--gray-background);
+
+    &__container {
+        width: 100%;
+        height: fit-content;
+        display: flex;
+        align-content: center;
+        /* flex-wrap: wrap; */
+        justify-content:space-between;
+        margin-bottom: $container-margin-bottom;
+    }
+
+    &__sticker {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: flex-end;
+        justify-content: space-between;
+        width: $sticker-width;
+        height: $sticker-height;
+    }
+
+    &__icon-box {
+        width: $icon-width;
+        height: $icon-height;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        justify-self: baseline;
+    }
+
+    &__description {
+        display: flex;
+        flex-direction: column;
+        align-content: center;
+    }
+
+    &__title {
+        @include abstracts.gradient-text;
+        align-self: center;
+        font-size: var(--larger-font-size);
+        padding-bottom: $title-padding-bottom;
+    }
+
+    &__paragraph {
+        text-align: center;
+        color: var( --standard-paragraph-color);
+        line-height: 1.47rem;
```

Comment:
Magic number spotted.

Reply:
Is that not something that can stay as it is? There isn't anything I can add to a variable to explain it better and within my code it has different values for many different areas.